test/unit/io.js

```diff
@@ -861,6 +861,74 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression shrinks test file', function (done) {
+    let startSize;
+    sharp(fixtures.inputUncompressedTiff)
+          .toBuffer((err, buffer, info) => {
```

Comment:
Only one `toBuffer` is supported but there are two here. Perhaps use [fs.statSync](https://nodejs.org/api/fs.html#fs_fs_statsync_path) to read `startSize`.